### ISSUE TITLE
Script is never run on pages with CSP and a service worker

### ISSUE THREAD

**Issue by wheany** [Tampermonkey/tampermonkey#956]
Tampermonkey scripts run only after a hard reload (shift-reload) on twitter (in the new layout). 

I think it's related to the service worker somehow where normal reload just fetches the old page from the cache, but a hard reload (Shift-reload button) actually fetches the page over the internet.

I have both the "Add Tampermonkey to the site's content security policy (CSP) if there is one" and "Add Tampermonkey to the site's content CSP" settings turned on.

## Expected Behavior
Script runs, printing "Tampermonkey service worker failure demo says hello!" in the console when visiting https://twitter.com/TheDailyOtter/status/1267748539670691840

## Actual Behavior
The following error, as well as several others, appears in the console
Security Error: Content at https://twitter.com/TheDailyOtter/status/1267748539670691840 may not load or link to moz-extension://4f14aa19-f3ee-4233-92c1-dfe360de16a5/js/content/inject.js.

## Specifications

- Firefox 76.0.1 (64-bit)
- OS: Windows 10

## Script

```js
// ==UserScript==
// @name         Tampermonkey service worker failure demo
// @match        https://twitter.com/TheDailyOtter/status/1267748539670691840
// @grant        none
// ==/UserScript==

console.log('Tampermonkey service worker failure demo says hello!');
```

**Comment by vaporwave9** [comment]
I can confirm this, today 3 Twitter scripts broke for me as well on Firefox when they enforced CSP with the new design.

The console logs listed CSP as the root cause. Hopefully Tampermonkey will be able to implement a way like Greasemonkey (which is in development hell since June 2019).

**Comment by derjanb** [comment]
The CSP part is a duplicate of #952 and the service worker part needs a workaround as mentioned at https://github.com/Tampermonkey/tampermonkey/issues/773#issuecomment-552951807

**Comment by vaporwave9** [comment]
So where can we track the issue, since all three issues were closed or is Firefox not supported anymore? The problem can be solved, as the execution of the affected scripts is possible in other userscript managers.

**Comment by derjanb** [comment]
> So where can we track the issue, since all three issues were closed

Closed? #952 and #773 are open. 👀

**Comment by vaporwave9** [comment]
Sorry, my brain doesn't work today. The mentioned issues were closed, not the 2 you linked.
I feel incredibly stupid now 🤦

**Comment by derjanb** [comment]
No problem. :-) BTW: I'll release a FF beta version with a workaround for #952 today.

**Comment by Owyn** [comment]
can confirm, userscripts won't run on twitter with firefox+TM, with no other extensions active
